### PR TITLE
Contributed To Task Management and Development Process

### PR DESCRIPTION
**Pull request by amir-alleyne**
- Added explanations of the SOLID Principles
- Inserted link with helpful examples of these Principles
- Added information regarding Task Management Software
- Provided an example in the form of Jira
- Added helpful links with tips and tricks and secret features of Jira

### CODE REVIEW

File: Topics/Teamwork.md [hunk 1]
```diff
@@ -5,3 +5,9 @@ Task management software is essential for organizations and individuals to impro
 
 # Examples of Common Task Management Software:
 Jira is a software development tool that allows teams to plan, track, and manage their projects effectively. It is a popular tool used by agile teams to improve their collaboration, visibility, and productivity. With Jira, teams can create tasks, assign them to team members, and track their progress in real-time. Jira also provides features such as dashboards, agile boards, and customizable workflows that help teams stay organized and focused on their goals. In addition, Jira integrates with other software development tools such as GitHub, Bitbucket, and Confluence, making it a powerful tool for software development teams.
+
+Resources for Jira:
+
+Beginner tutorial for Jira - https://www.youtube.com/watch?v=nHuhojfjeUY 
+
```
Comment (MicheleMassa802) [inline Topics/Teamwork.md:12]: Add extra article/written-type resource here (refer to PR comment for more details).

File: Topics/Teamwork.md [hunk 2]
```diff
@@ -1 +1,7 @@
 ## Resources for Teamwork
+
```
Comment (MicheleMassa802) [inline Topics/Teamwork.md:2]: Possibly move and adapt this section over to Task_Management_Software.md to accompany your other contribution, or add a reference to it for a reader of this file to quickly access.

**Comment by MicheleMassa802** [comment]
This seems like an overall solid contribution (no pun intended). 

Under your SOLID principles contribution, I like that seeing as your source was a YouTube video, you paraphrased some of the main takeaways from each principle, providing a nice and accurate overview of the topic, such that anyone interested in it can dive further into it by going to the video.

For your other contributions I see that you have added them to both the Teamwork and Task Management files, and while you fully justified how Jira can be used in a team environment, I think I'd prefer to see it fully on Task_Management_Software.md, as to not split the information on this tool across different files (by this I mean, taking the description that you put in Teamwork.md and adapting it into the Task_Management_Software.md description). 

Alternatively, if you really want to keep the information split across the two files in an effort to have all of your contributions perfectly match the purpose of the files, I would like to see you include a link/reference on the Teamwork.md submission to the Task_Management_Software.md file section, as it would allow anyone looking over the teamwork file to easily transition into how to use the tool that you are recommending for their team.

Additionally, I would appreciate if you could include at least one article-type resource for your Jira contribution over on Task_Management_Software.md, because while the linked videos are good, it would be nice to have an option that allows the reader to overview the topic a little faster than an hour long video before they decide to commit to the tool.

Apart from that the submission seems pretty good and clean. If you can resolve these issues, I'll gladly merge your PR.

**Comment by amir-alleyne** [comment]
> 

Thank you Michele! I completely missed that it was the wrong file. It was intended for Task Management not teamwork! Thank you for catching that. 
I also added a helpful written article on how Jira workflows work to allow readers to gether information quickly

**Comment by MicheleMassa802** [comment]
I like the Atlassian article you linked to the Jira contribution, it's short and resolves the issue I had with the length of the other resources that come in the form of videos.

The incorporation of the team-side and organization aspects discussion of Jira into the Task_Management_Software.md (and out of Teamwork.md) is more than sufficient and solves the concern I previously had. 

The submission looks clean and provides a perfect amount of information on the topics it discusses (enough to spark a reader's curiosity on the tool/concepts, while leading them to the correct and proper resources to deepen their understanding if they are interested/find it useful).

I am then merging your PR, nice contributions all around.

**Pull request merged**